meteor/client/ui/RundownList/RundownListItem.tsx

```diff
@@ -264,6 +273,16 @@ export const RundownListItem = translateWithTracker<IRundownListItemProps, {}, I
 					// rundown ids can start with digits, which is illegal for HTML id attributes
 					const htmlElementId = `${HTML_ID_PREFIX}${unprotectString(rundown._id)}`
 
+					const showStyleLabel =
+						this.props.showStyleVariant &&
+						this.props.showStyleBase &&
+						this.props.showStyleVariant.name !== this.props.showStyleBase.name
+							? t('{{showStyleVariant}} - {{showStyleBase}}', {
```

Comment:
I'd argue that running this through the translator is unnecessary, given that there's no content that actually can be translated :)

Reply:
I'm running these kinds of strings through the translator system to be able to support non-European languages - in some languages the separator should be other than an en-dash - Japanese would use a "〜" and Hebrew would use a 0x05be character (an en-dash, but in a different position relative to the baseline). I'm not religious about it, but I try to use that system whenever possible - for consistency - when concatenating UI strings.